tools/cachemgr.cc

```diff
@@ -1251,7 +1240,7 @@ check_target_acl(const char *hostname, int port)
         fp = fopen(DEFAULT_CACHEMGR_CONFIG, "r");
 
     if (fp == NULL) {
-#ifdef CACHEMGR_HOSTNAME_DEFINED
+#if defined(CACHEMGR_HOSTNAME)
```

Comment:
AFAICT, this seemingly equivalent replacement actually makes the condition always true because `CACHEMGR_HOSTNAME` is always defined _here_ (unlike the code where misnamed `CACHEMGR_HOSTNAME_DEFINED` was defined before this PR).

P.S. Sorry for a few stand-alone comments. I ran out of time to do a full review, but I think you are making great progress!